pkg/api/v1beta3/types.go

```diff
@@ -814,6 +814,10 @@ type PodSpec struct {
 	// NodeSelector is a selector which must be true for the pod to fit on a node
 	NodeSelector map[string]string `json:"nodeSelector,omitempty" description:"selector which must match a node's labels for the pod to be scheduled on that node"`
 
+	// ServiceAccount is the name of the ServiceAccount to use to run this pod
+	// The pod will be allowed to use secrets referenced by the ServiceAccount
+	ServiceAccount string `json:"serviceAccount" description:"name of the ServiceAccount to use to run this pod"`
```

Comment:
As described here:
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/api-conventions.md#strategic-merge-patch

It should be possible to merge lists of scalars, which would be deduped and merged. No merge key required.